Source/EasyNetQ/AdvancedMessagePolymorphism/AdvancedPolymorphismPublishExchangeDeclareStrategy.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
+using EasyNetQ.Internals;
+using EasyNetQ.Producer;
+using EasyNetQ.Topology;
+
+
+namespace EasyNetQ.AdvancedMessagePolymorphism
+{
+    public class AdvancedPolymorphismPublishExchangeDeclareStrategy : IPublishExchangeDeclareStrategy
```

Comment:
Can we think of a more descriptive name instead of 'Advanced*', something like MultipleExchange*?